src/Copy/AboutUs.elm

```diff
@@ -27,7 +27,7 @@ Ivan and Alfie had key development roles on PlaceCal, an award-winning community
 emma : Model.ProfileInfo
 emma =
     { section = ContentAndDesign
-    , name = "Emma"
+    , name = "Emma Charleston"
```

Comment:
Stuart and I decided to leave last names out. I'll check other opinions in Zulip.